api/cloudcontroller/ccv3/metadata.go

```diff
@@ -34,6 +34,12 @@ func (client *Client) UpdateResourceMetadata(resource string, resourceGUID strin
 			Body:        bytes.NewReader(metadataBtyes),
 			URIParams:   map[string]string{"app_guid": resourceGUID},
 		})
+	case "buildpack":
+		request, err = client.newHTTPRequest(requestOptions{
+			RequestName: internal.PatchBuildpackRequest,
+			Body:        bytes.NewReader(metadataBtyes),
```

Comment:
Can we correct the spelling of `metadataBytes` while we're making this change.

Reply:
I've got something on now, but will do. Thanks for catching that

Reply:
I'll have to work on that test on Friday (PDT). Want to get all the combinations right.